internal/db/queries.sql

```diff
@@ -103,3 +103,18 @@ DELETE FROM public.repos WHERE repo_import_id = $1::uuid AND NOT(repo = ANY($2::
 
 -- name: CleanOldRepoSyncQueue :exec
 SELECT mergestat.simple_repo_sync_queue_cleanup($1::INTEGER);
+
+-- name: GetAllReposId :many
+SELECT id FROM public.repos WHERE repo_import_id = @importID::uuid
+;
+
+-- name: AddingNewDefaultSync :one
+SELECT mergestat.insert_default_sync(@repoID::uuid,@syncType::text);
```

Comment:
Yes , I have the wrong idea of enqueuing  the newly  added syncs so I needed  the boolean from the function. 
Now It can be a online insert with do nothing on conflict